File: Ass1_Q2.cpp
#include <stdio.h>

int main() {
    int s=1,m=0,n=0,t=0;
    printf("Enter any integer:");
    scanf("%d", &n);
   if (n >= 0){
       int i;
        for(i=1;i<=n;++i){
            s =s*i;
        }
        printf("factorial:%d", s);
   }
   else{
       
        printf("please choose positive integers to find factorial");
    }
    return 0;
}

File: Ass1_Q2.py
def factorial(n):
    if n>=1:
        if n==1:
            return n
        else:
            return n * factorial(n-1)
    else:
        print("please take positive integers")
n = int(input("Enter an integer: "))
print("factorial of the number =" + str(factorial(n)))
